src/metaschema/oscal_metadata_metaschema.xml

```diff
@@ -54,7 +54,12 @@
                         <enum value="alternate">The link identifies an alternative location or format for this resource. Defined by <a href="https://html.spec.whatwg.org/multipage/links.html#linkTypes">the HTML Living Standard</a></enum>
                         <enum value="predecessor-version">This link identifies a resource containing the predecessor version in the version history. Defined by  <a href="https://tools.ietf.org/html/rfc5829">RFC 5829</a>.</enum>
                         <enum value="successor-version">This link identifies a resource containing the predecessor version in the version history. Defined by <a href="https://tools.ietf.org/html/rfc5829">RFC 5829</a>.</enum>
+                        <enum value="version-history">This link identifies a resource containing the version history of this document. Defined by <a href="https://tools.ietf.org/html/rfc5829">RFC 5829</a>.</enum>
                     </allowed-values>
+                    <!-- 
```

Comment:
I would like to discuss the feasibility and usefulness of this.

Reply:
I was asked to convert this into an issue for future development, it is not seen as an immediate enhancement or fix to improve the maintenance of the current model in this area.

Reply:
I think we can only check for if the last-modified is >= the latest revision.